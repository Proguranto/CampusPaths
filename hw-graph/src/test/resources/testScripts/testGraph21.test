# Tests acyclic graph creation/viewing of a graph
# with 2 node, 1 edge

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 e1

ListNodes graph1

ListChildren graph1 n1
ListChildren graph1 n2
