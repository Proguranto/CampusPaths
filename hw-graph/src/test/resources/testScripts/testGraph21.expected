# Tests acyclic graph creation/viewing of a graph
# with 2 node, 1 edge

created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge e1 from n1 to n2 in graph1

graph1 contains: n1 n2

the children of n1 in graph1 are: n2(e1)
the children of n2 in graph2 are:
